LICENSE

```diff
@@ -2,14 +2,11 @@ The MIT License (MIT)
 
 Copyright (c) 2016 Yuki Hattori (yukihattori1116@gmail.com)
 
-This software also uses portions of the KaTeX and underscore.js projects, which are
```

Comment:
`underscore` is used by KaTeX which is why it is included there, i.e. see https://github.com/Khan/KaTeX/blob/master/LICENSE.txt 

Don't really know if `marp` needs to state it in the license file as I'm not a lawyer,, but that is why the text was put there in the KaTeX PR.